server/utils/urlAccessByStatus.ts

```diff
@@ -49,6 +49,18 @@ const allowedPaths = [
     status: 'RECALLED',
     allowed: ['/licence/create/id/(\\d)*/check-your-answers.*', '/licence/view/id/(\\d)*/.*'],
   },
+  {
+    status: 'VARIATION_IN_PROGRESS',
+    allowed: ['/licence/create/.*', '/licence/vary/.*'],
+  },
+  {
+    status: 'VARIATION_SUBMITTED',
+    allowed: ['/licence/vary/.*'],
+  },
```

Comment:
Do VARIATION_SUBMITTED or VARIATION_APPROVED licences ever need to go via `/licence/created/check-your-answers` or are these entirely dealt with within the /vary routes?

Reply:
These statuses cannot go to the check your answers page. If they need editing they transition to VARIATION_IN_PROGRESS first